themes/vpge/lib/js/gradgrow-filters/src/gradgrow-filters.island.tsx

```diff
@@ -144,16 +144,17 @@ const GradGrowFilters = () => {
           />
         </div>
       </FilterContainer>
-      <div style={{display: "flex", gap: "20px"}}>
-        <button onClick={submit}>Apply Filters
-        </button>
-
-        {(selectedOpportunity || selectedTime || selectedCompetency) &&
-          <button onClick={reset}>
-            Reset Filters
-          </button>
-        }
-      </div>
+      <button onClick={submit} style={{width: "fit-content"}}>Apply Filters
+      </button>
+
+      {(selectedOpportunity || selectedTime || selectedCompetency) &&
+        <a onClick={reset} style={{cursor: "pointer", fontSize: "2rem"}}>
```

Comment:
Since you are changing this to an `<a>` tag, you can remove the `onClick` (and the reset function above. then you can just use `href={window.location.pathname}`

Reply:
Thanks, updated.